protos/format.proto

```diff
@@ -153,6 +164,11 @@ message DataFragment {
 
   // File that indicates which rows, if any, should be considered deleted.
   DeletionFile deletion_file = 3;
+
+  // Number of original rows in the fragment, not counting deletions. To compute
+  // the current number of rows, subtract `deletion_file.num_deleted_rows` from
+  // this value.
+  uint64 fragment_length = 4;
```

Comment:
It is not clear that it means physical rows (instead of logical rows). 

Could we change to another name?

Btw, do we plan to maintain backward compatibility?

Reply:
> Could we change to another name?

I thought `fragment_length` was a good name to differentiate from `num_rows`. Is there a better name you have in mind? `physical_length` or something like that?

> Btw, do we plan to maintain backward compatibility?

Yes, this PR is designed to support backwards compatibility, though it does migrate any tables it writes to the new scheme.

Reply:
Maybe just `physical_rows` ? `fragment_length` is somewhat similar to `Fragment::len()` in semantic , to me.

Reply:
Ahh I see where you get that from. I can change to `physical_rows`.